example/ios/RiveReactNativeExample.xcodeproj/project.pbxproj

```diff
@@ -724,7 +734,7 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.example.rivereactnative;
+				PRODUCT_BUNDLE_IDENTIFIER = gordon.example.rivereactnative;
```

Comment:
Probably fine, just calling this out as a thing that's been committed.

Reply:
Ah I forgot this, will remove